app/Services/Entity/MultiEditingService.php

```diff
@@ -73,7 +78,17 @@ public function isEditing(): bool
     public function edit(): self
     {
         $model = new EntityUser();
-        $model->entity_id = $this->entity->id;
+        if ($this->model->getTable() == 'entity_notes') {
```

Comment:
If we change the table name, all this code will need to be manually updated. Instead you can use `$this->model instanceof EntityNote` to check the object's class.